frontend/wallet/package.json

```diff
@@ -42,7 +42,7 @@
     "dist": "yarn run build && yarn run pack && build",
     "dev-real": "concurrently --kill-others 'bsb -make-world -w' 'fpack watch --development ./lib/js/src/render/Index.js' 'electron ./lib/js/src/main/App.js'",
     "dev": "GRAPHQL_BACKEND=faker yarn dev-real",
-    "regen-sdl": "node generate-schema.js http://localhost:8080/graphql > schema.graphql"
+    "regen-sdl": "node generate-schema.js http://0.0.0.0:8080/graphql > schema.graphql"
```

Comment:
I don't know what's going on here

Reply:
I talked to @Schmavery. And he said this was fine.

Reply:
If that URL ever gets used to actually make a connection, it shouldn't be fine, because 0.0.0.0 is not a routable address...